site/content/en/docs/formats/brats.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'BraTS'
+linkTitle: 'BraTS'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original BraTS dataset is available
+[here](https://www.med.upenn.edu/sbia/brats2018/data.html).
+
+Supported annotation types:
+- `Mask`
+
```

Comment:
Please reflect what is the frame data layout in the parsed representation and how to interpret annotations read.

Reply:
Added an information about it.